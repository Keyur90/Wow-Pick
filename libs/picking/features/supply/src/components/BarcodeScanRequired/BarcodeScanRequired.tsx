import React, { useState } from 'react';
import { PopUpModal } from '@rf2/ui';

interface BarcodeScanRequiredProps {
  isOpen: boolean;
  onClose: () => void;
}

const BarcodeScanRequired: React.FC<BarcodeScanRequiredProps> = ({ isOpen, onClose }) => {
  return (
    <PopUpModal
      isOpen={isOpen}
      onOkClick={onClose}
      bodyText="Barcode scan required before won’t scan is enabled, please scan…"
      headerText="Won't Scan"
      closeOrOkText="Close"
    />
  );
};

export { BarcodeScanRequired };
