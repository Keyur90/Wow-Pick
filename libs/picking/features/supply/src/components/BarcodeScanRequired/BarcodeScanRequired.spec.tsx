import React from 'react';
import { render } from '@testing-library/react';
import { BarcodeScanRequired } from './BarcodeScanRequired';

test('renders BarcodeScanRequired', () => {
  const { container } = render(<BarcodeScanRequired isOpen={true} onClose={null} />);

  expect(container).toMatchSnapshot();
});
