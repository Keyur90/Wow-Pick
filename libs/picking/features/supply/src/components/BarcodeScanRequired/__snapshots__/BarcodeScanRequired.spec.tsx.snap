// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders BarcodeScanRequired 1`] = `
<div
  aria-hidden="true"
/>
`;
